test/lambdaisland/clj_diff/miller_test.clj

```diff
@@ -1,7 +1,7 @@
 (ns lambdaisland.clj-diff.miller-test
-  (:require [clojure.test :refer :all]
+  (:require [clojure.test :refer [deftest is are]]
             [lambdaisland.clj-diff.core :as core]
-            [lambdaisland.clj-diff.miller :refer :all]))
+            [lambdaisland.clj-diff.miller :refer [diff ses]]))
```

Comment:
This was necessary since there was a test in this namespace with the same name as an imported var, so it caused  a conflict.